frontend/src/metabase-lib/v1/expressions/complete/fields.ts

```diff
@@ -15,12 +15,17 @@ export type Options = {
   expressionIndex?: number;
 };
 
-export function suggestFields({ query, stageIndex, expressionIndex }: Options) {
-  const columns = Lib.expressionableColumns(
-    query,
-    stageIndex,
-    expressionIndex,
-  )?.map((column) => {
+export function suggestFields({
+  query,
+  stageIndex,
+  expressionIndex,
+  expressionMode,
+}: Options) {
+  const f =
+    expressionMode === "aggregation"
+      ? Lib.aggregableColumns
+      : Lib.expressionableColumns;
```

Comment:
Can we extract this into a helper function since it's being used in a couple of places.

Reply:
Yes, please! :-)